### PR TITLE
[@tools-website] Home and faucet page UI fixes

### PR DESCRIPTION
**Pull request by KristinaSpasevska**

### CODE REVIEW

File: packages/apps/tools/src/components/Common/Layout/partials/Sidebar/MenuLinkButton.tsx [hunk 1]
```diff
@@ -1,41 +1,33 @@
...
   href?: string;
   active?: boolean;
+  target?: string;
 }
 
 export const MenuLinkButton: FC<IMenuLinkButtonProps> = ({
   active,
   title,
   href,
+  target = '_self',
```
Comment (sanderlooijenga) [inline packages/apps/tools/src/components/Common/Layout/partials/Sidebar/MenuLinkButton.tsx:19]: `target` isn't used anywhere?

File: packages/apps/tools/src/utils/persist.ts [hunk 2]
```diff
@@ -9,39 +9,10 @@ export const encode = (value: unknown) =>
...
 export const getItem = (key: string) => {
-  try {
-    const cookie = Cookies.get(getName(key));
-    return cookie !== undefined ? parse(cookie) : undefined;
-  } catch (e) {
-    return undefined;
-  }
-};
-
-export const deleteItem = (key: string) => {
```
Comment (sanderlooijenga) [inline packages/apps/tools/src/utils/persist.ts:25]: Out of curiosity; Why are these deleted?

File: packages/apps/tools/src/utils/persist.ts [hunk 3]
```diff
@@ -9,39 +9,10 @@ export const encode = (value: unknown) =>
...
-    {} as { [key: string]: string },
-  );
-};
-
-export const purge = () => {
-  const cookies = getItems();
-  Object.keys(cookies).forEach((key) => {
-    deleteItem(key);
-  });
+  const cookie = Cookies.get(getName(key));
```
Comment (sanderlooijenga) [inline packages/apps/tools/src/utils/persist.ts:16]: Why no longer is `try … catch`?

File: packages/apps/tools/src/components/Global/AccountNameField/index.tsx [hunk 4]
```diff
@@ -43,44 +31,13 @@ export const AccountNameField: FC<IAccountNameFieldProps> = ({
...
-  } = useWalletConnectClient();
-
-  useDidUpdateEffect(() => {
-    if (session && accounts?.length) setMode('select');
-    else setMode('input');
-  }, [session, accounts]);
+  const { selectedAccount, setSelectedAccount } = useWalletConnectClient();
 
   const lookup = {
-    select: (
```
Comment (sanderlooijenga) [inline packages/apps/tools/src/components/Global/AccountNameField/index.tsx:64]: Should we really delete this? 🤔

File: packages/apps/tools/src/components/Common/Layout/partials/Sidebar/styles.css.ts [hunk 5]
```diff
@@ -68,12 +51,16 @@ export const gridMiniMenuLinkButtonStyle = style([
     padding: '$5',
     textAlign: 'left',
     fontWeight: '$bold',
+    fontSize: '$sm',
   }),
   {
-    transition: 'all 0.1s ease',
     width: '100%',
-    borderBottom: `1px solid ${vars.colors.$borderDefault}`,
+    borderBottom: `1px solid #E3E1E5`,
```
Comment (sanderlooijenga) [inline packages/apps/tools/src/components/Common/Layout/partials/Sidebar/styles.css.ts:58]: I guess these fixed colour values are missing from the design tokens?

File: packages/apps/tools/src/utils/persist.ts (hunk 2 above)
Comment (KristinaSpasevska) [reply packages/apps/tools/src/utils/persist.ts:25]: They were not used anymore

File: packages/apps/tools/src/components/Global/AccountNameField/index.tsx (hunk 4 above)
Comment (KristinaSpasevska) [reply packages/apps/tools/src/components/Global/AccountNameField/index.tsx:64]: Didn't want to leave it commented out for later, but if you think that's better I can bring this back (for now we don't have the accounts from the wallet)

File: packages/apps/tools/src/utils/persist.ts (hunk 3 above)
Comment (KristinaSpasevska) [reply packages/apps/tools/src/utils/persist.ts:16]: trying out something, but will revert this

File: packages/apps/tools/src/components/Common/Layout/partials/Sidebar/styles.css.ts (hunk 5 above)
Comment (KristinaSpasevska) [reply packages/apps/tools/src/components/Common/Layout/partials/Sidebar/styles.css.ts:58]: yes, I got them from Isa for the time being until they add all the new tokens in the ui library

File: packages/apps/tools/src/components/Global/AccountNameField/index.tsx (hunk 4 above)
Comment (sanderlooijenga) [reply packages/apps/tools/src/components/Global/AccountNameField/index.tsx:64]: I mean I guess we can always get them back by referencing this PR, so it's fine like this 👍

File: packages/apps/tools/src/utils/persist.ts (hunk 2 above)
Comment (sanderlooijenga) [reply packages/apps/tools/src/utils/persist.ts:25]: Were they ever used? I think in this case these could stay in, as they could potentially be used later. What do you think?

Comment (KristinaSpasevska) [reply packages/apps/tools/src/utils/persist.ts:25]: Not a problem to retrieve them

Comment (sanderlooijenga) [reply packages/apps/tools/src/utils/persist.ts:25]: If you can run the formatter then that would be great, I'm approving in the meantime 😄

**Review by sanderlooijenga — APPROVED** [review]
